src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, setTab }) => {
+  const selectedTab = tabs.find(tab => (tab.id === selectedTabId)) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn(
+                { 'is active': selectedTab === tab.id },
```

Comment:
The selected tab is not highlighted. Use the correct className for it, please 
![image](https://github.com/mate-academy/react_tabs-js/assets/99403198/d8028d99-c485-4d61-a5d5-8da940b895e8)
![image](https://github.com/mate-academy/react_tabs-js/assets/99403198/9abbc265-e986-4623-914e-75df8ef13790)